Remotion/Development/Web/ResourceHosting/ResourceVirtualPathProvider.cs

```diff
@@ -170,21 +172,21 @@ public override CacheDependency GetCacheDependency (string virtualPath, IEnumera
         if (IsMappedPath (appRelative))
         {
           if (FileExists (appRelative))
-            str = GetResourceVirtualFile (appRelative).PhysicalPath;
+            str = GetResourceVirtualFile (appRelative).PhysicalPath!; // TODO: Assert not null
```

Comment:
assertion, since we check first.

Reply:
In this case no assertion would be necessary right?

Reply:
hmm...the age old problem, should we assert our contracts if the compiler can't do it? Here, I think we should, but probably just before adding it to the string collection a couple lines down. If we allow null to pass through here, we never find the bug again.